inc/functions_post.php

```diff
@@ -401,64 +401,81 @@ function build_postbit($post, $post_type=0)
 		}
 
 		// Display profile fields on posts - only if field is filled in
-		if(is_array($profile_fields))
```

Comment:
> `is_array` returns false if the variable isn't set anyways.

That is the reason you will be checking if it is set before the check, set it if false. This will make profile fields work on all postbits and not assume that it should only work on threads (ignoring quick reply at the moment anyways).

Just one potential fix to consider.